glutton/glutton_server.go

```diff
@@ -83,6 +93,20 @@ func main() {
 	}
 	defer filePointer.Close()
 
-	go tcpListener(filePointer)
-	udpListener(filePointer)
+	//channel for tcp logging
+	tcpChannel := nbc.New()
+	//channel for udp logging
+	udpChannel := nbc.New()
+
+	fmt.Println("Initializaing TCP connections tracking...")
+	go Monitor_Connections("tcp", tcpChannel)
+
+	fmt.Println("Initializaing UDP connections tracking...")
+	go Monitor_Connections("udp", tcpChannel)
```

Comment:
Yes.. I am working on udp. I make this PR for showing how I am handling tcp. 
I will send PR soon after solving udp and all the errors you highlighted here.